contents/tutorials/frontend-vs-backend-group-analytics.md

```diff
@@ -0,0 +1,156 @@
+---
+title: The guide to frontend vs backend group analytics implementations
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-11-09
+featuredImage: ../images/tutorials/banners/frontend-vs-backend-group-analytics.png
+topics: ["configuration"]
+---
+
+Group analytics is a powerful feature for understanding how groups such as organizations, customers, and companies use your product as a unit. It provides a new level of analysis between individual users and all your users.
+
+When implementing group analytics, there are some differences to look out for between the frontend implementation and the backend one. This tutorial goes over why there are differences and how to implement group analytics on the frontend compared to the backend.
+
+## Why are there differences
+
+Before we go into the implementation, it is useful to understand **why** there are differences between the frontend and backend. 
+
+A group is generally defined as a collective of people with a shared identity. At PostHog, we think about (and store) everything as **events**. This means a group for us is a set of events related to a group identity (and properties). People (in PostHog) are the same, a set of events related to a person identity (and properties). 
```

Comment:
> A group is generally defined as a collective of people with a shared identity.

Hmm, this isn't correct. A group can be both a collective of people, OR, multiple groups can belong to the same person, which is why groups are based on events.

To give an example: Organisation is a group, but so is a paymentID.

Every time a person makes a payment, a new 'group' paymentID is generated. You can see how paymentID is not a collective of people, but a collective of paymentIDs makes a person. Makes sense?

Reply:
I'm trying to say in this paragraph that how normal people think of a group is a collective of people, but at PostHog we think of a group as a collective of events.

I get that a person can belong to multiple groups, but I don't understand how that changes the "general" definition of a group. The people of an organization share an identity, so are a group. The events share a paymentID so they are a group too. 

Does changing the sentence to "A group is generally defined as a collective of people or things with a shared identity." make it more correct?

Reply:
Ahhh, I see, what you're defining is a 'group' in english, not a posthog group. It adds more confusion honestly, since I'd _hope_ people reading this know what an english 'group' is already?

I'd make what you're intending to say explicit, or just get rid of the first line? Thoughts?

Reply:
```suggestion
At PostHog, we think about (and store) everything as **events**. This means a group for us is a set of events related to a group identity (and properties). People (in PostHog) are the same, a set of events related to a person identity (and properties). 
```

Reply:
Also, what is the intention behind `(and properties)` ? 

"This means a group for us is a set of events with a common group identity. People in PostHog are the same, a set of events with a common person identity."

Reply:
Good points, I'll make both changes.